### ISSUE TITLE
Could not find active fragment with index -1

### ISSUE THREAD

**Issue by litrik** [Pushwoosh/pushwoosh-android-sdk#153]
Versions
 - com.pushwoosh:pushwoosh:6.1.0
 - com.pushwoosh:pushwoosh-firebase:6.1.0

I have seen the following crash arrive in Crashlytics:

```
Caused by java.lang.IllegalStateException: Could not find active fragment with index -1
       at android.app.FragmentManagerImpl.restoreAllState(FragmentManager.java:2833)
       at android.app.FragmentController.restoreAllState(FragmentController.java:140)
       at android.app.Activity.onCreate(Activity.java:1044)
       at com.pushwoosh.inapp.view.WebActivity.onCreate()
       at com.pushwoosh.inapp.view.RichMediaWebActivity.onCreate()
       at android.app.Activity.performCreate(Activity.java:7159)
       at android.app.Activity.performCreate(Activity.java:7150)
       at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1272)
       at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3001)
       at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3156)
       at android.app.ActivityThread.handleRelaunchActivityInner(ActivityThread.java:4973)
       at android.app.ActivityThread.handleRelaunchActivity(ActivityThread.java:4883)
       at android.app.servertransaction.ActivityRelaunchItem.execute(ActivityRelaunchItem.java:69)
       at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
       at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1864)
       at android.os.Handler.dispatchMessage(Handler.java:106)
       at android.os.Looper.loop(Looper.java:205)
       at android.app.ActivityThread.main(ActivityThread.java:6991)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:884)
```

**Comment by wfhm** [comment]
@litrik,

We could not reproduce the issue so far. By any chance could you please share a simple reproducer and describe the steps of how to reproduce the issue on our side?

**Comment by litrik** [comment]
I haven't been able to reproduce this either. It only appeared in our Crashlytics logs. I was hoping the stack trace might be sufficient for you to investigate this further.

**Comment by wfhm** [comment]
@litrik,

Fixed with 6.1.1
